site/content/3.10/arangograph/monitoring-and-metrics.md

```diff
@@ -3,25 +3,75 @@ title: Monitoring & Metrics in ArangoGraph
-[Prometheus](https://prometheus.io/)
-compatible format.
-You can use these metrics to gather detailed insights into the current
-and previous states of your deployment.
-Once metrics are collected by Prometheus, you can inspect them using tools
-such as [Grafana](https://grafana.com/oss/grafana/).
+The ArangoGraph Insights Platform provides integrated charts, metrics, and logs
+to help you monitor your deployment. This allows you to track your deployment's
+performance, resource utilization, and its overall status.
+
+The key features include:
+- **Built-in monitoring**: Get immediate access to monitoring capabilities for
+  your deployments without any additional setup.
+- **Chart-based metrics representation**: Visualize the usage of the DBServers
+  and Coordinators over a selected timeframe.
+- **Integration with Prometheus and Grafana**: Connect your metrics to Prometheus
+  and Grafana for in-depth visualization and analysis.
 
 To get started, select an existing deployment from within a project and
 click **Monitoring** in the navigation. 
 
 ![ArangoGraph Monitoring tab](../../images/arangograph-monitoring-tab.png)
 
+## Build-in monitoring and metrics
+
+### In the **Servers** section
+
+The **Servers** section offers an overview of the DBServers, Coordinators,
+and Agents used in your deployment. It provides essential details such as each
+server's ID and type, the running ArangoDB version, as well as their memory,
+CPU, and disk usage.
+
+In case you need to perform a restart on a server, you can do so by using the
+**Gracefully restart this server** action button. This shuts down all services
+normally, allowing ongoing operations to finish gracefully before the restart
+occurs.
+
+Additionally, you can access detailed logs via the **Logs** button. This allows
+you to apply filters to obtain logs from all server types or select specific ones
+(i.e. only Coordinators or only DBServers) within a timeframe. To download the
+logs, click the **Save** button.
+
+![ArangoGraph Monitoring Servers](../../images/arangograph-monitoring-servers.png)
+
+### In the **Metrics** section
+
+The **Metrics** section displays a chart-based representation depicting the
+resource utilization of DBServers and Coordinators within a specified timeframe.
+
+You can select one or more DBServers and choose **CPU**, **Memory**, or **Disk**
+to visualize their respective usage. The search box enables you to easily find
+a server by its ID, particularly useful when having a large number of servers
+or when needing to quickly find a particular one among many.
+
+Similarly, you can repeat the process for Coordinators to see the **CPU** and
+**Memory** usage.
+
+![Arangograph Monitoring Metrics Chart](../../images/arangograph-monitoring-metrics-chart.png)
+
+## Connect with Prometheus and Grafana
+
+The ArangoGraph Insights Platform provides metrics for each deployment in a 
+[Prometheus](https://prometheus.io/) compatible format.
```

Comment:
Would be better to rewrite as this is an external link for which we show an icon on the right side
```suggestion
[Prometheus](https://prometheus.io/)-compatible format.
```